### DISCUSSION TITLE
Bicopter VTOL (like Tom Stanton)

### DISCUSSION THREAD

**Discussion by Flokmayer** [iNavFlight/inav#9870] [Q&A] [answered]
Is it possible to setup a VTOL like Tom Stanton? 
For those who have not seen the video: In Vertical mode it should act like a normal Bicopter, but in forward flight its Yaw-axis should be controlled by differential thrust, the Roll-axis by tilting the servos and the pitch-axis by a elevator

**Comment by DzikuVx** [comment, marked as answer]
Yes, this is a standard VTOL mode and INAV supports since version 7. However, please rememeber Tom built a couple of different VTOLS including with very custom control mechanism and for that a custom solution would be required too.
But as long as VTOL is build with "standard" components, INAV can handle it
Please refer to https://github.com/iNavFlight/inav/blob/master/docs/VTOL.md

**Reply by Flokmayer** [reply]
Thank you for your answer, I am excited to build a VTOL and very happy to hear that it is possible.
Best regards.

**Comment by sensei-hacker** [comment]
A tip for setting this mixer up since it can get confusing:

Start with no lines in your motor or servo mixers.
Choose ONE output (motor or servo), and ONE axis such as pitch.  Add one mixer line, for that one motor, and set only the pitch value. Get that one motor working the way you want on pitch only,  without worrying about anything else.  Then do roll on that one motor.  One output at a time, one axis at a time. Get each one right before adding a line for the next item output.

Here's the part that bites people - while working on each value, don't go back and change the earlier lines / values. They were already done and working correctly.

**Reply by Flokmayer** [reply]
Thank you for the tip, I have no prior experience using INAV (only used Betaflight). I hope it won't take to long getting everything working.
